ckanext/stadtzh_losdharvest/profiles.py

```diff
@@ -112,6 +112,11 @@ def parse_dataset(self, dataset_dict, dataset_ref):
         ] = "http://creativecommons.org/licenses/by/3.0/"
         dataset_dict["license_title"] = "CC-BY 3.0"
 
+        # rights
+        dataset_dict["legalInformation"] = self._get_rights_for_dataset_ref(
```

Comment:
I foolishly merged before making this comment, sorry. Shouldn't this be `legal_information` in snake_case?

Reply:
Oh, no, I checked in the code and I see that was correct. Who knows why we did it that way!